terraform/cloudfoundry/routers.tf

```diff
@@ -51,3 +51,93 @@ resource "aws_elb" "ssh-proxy-router" {
     lb_protocol = "tcp"
   }
 }
+
+resource "aws_elb" "ingestor_elb" {
+  name = "${var.env}-ingestor-elb"
+  subnets = ["${split(",", var.infra_subnet_ids)}"]
+  idle_timeout = "${var.elb_idle_timeout}"
+  cross_zone_load_balancing = "true"
+  internal = "true"
+  security_groups = [
+    "${aws_security_group.ingestor_elb.id}",
+  ]
+
+  health_check {
+    target = "TCP:5514"
+    interval = "${var.health_check_interval}"
+    timeout = "${var.health_check_timeout}"
+    healthy_threshold = "${var.health_check_healthy}"
+    unhealthy_threshold = "${var.health_check_unhealthy}"
+  }
+  listener {
+    instance_port = 5514
+    instance_protocol = "tcp"
+    lb_port = 5514
+    lb_protocol = "tcp"
+  }
+  listener {
+    instance_port = 2514
+    instance_protocol = "tcp"
+    lb_port = 2514
+    lb_protocol = "tcp"
+  }
+}
+
+resource "aws_elb" "kibana_elb" {
+  name = "${var.env}-cf-kibana-elb"
+  subnets = ["${split(",", var.infra_subnet_ids)}"]
+  idle_timeout = "${var.elb_idle_timeout}"
+  cross_zone_load_balancing = "true"
+  security_groups = [
+    "${aws_security_group.web.id}",
+  ]
+
+  health_check {
+    target = "TCP:5601"
+    interval = "${var.health_check_interval}"
+    timeout = "${var.health_check_timeout}"
+    healthy_threshold = "${var.health_check_healthy}"
+    unhealthy_threshold = "${var.health_check_unhealthy}"
+  }
+  listener {
+    instance_port = 5601
+    instance_protocol = "tcp"
+    lb_port = 80
```

Comment:
From meatspace: it's significant more work to generate a new cert and upload it to IAM in the pipeline, so we've decided to not introduce the ELB in this story and will continue accessing it using an SSH tunnel. I'm a bit indifferent about whether we keep two Kibana instances or not - Jonty can decide. We'll create another story to replace the SSH tunnels for Logs and Metrics.